re.sonny.Workbench.Devel.json

```diff
@@ -5,11 +5,21 @@
   "sdk": "org.gnome.Sdk",
   "sdk-extensions": [
     "org.freedesktop.Sdk.Extension.vala",
+    "org.freedesktop.Sdk.Extension.rust-stable",
     "org.gnome.Sdk.Docs"
   ],
   "build-options": {
-    "append-path": "/usr/lib/sdk/vala/bin",
-    "append-ld-library-path": "/usr/lib/sdk/vala/lib"
+    "append-path": "/usr/lib/sdk/rust-stable/bin:/usr/lib/sdk/llvm14/bin:/usr/lib/sdk/vala/bin",
+    "append-ld-library-path": "/usr/lib/sdk/vala/lib",
+    "build-args": ["--share=network"],
+    "env": {
+      "CARGO_REGISTRIES_CRATES_IO_PROTOCOL": "sparse",
```

Comment:
This is no longer needed on latest cargo versions

Reply:
Good catch, I wanted to remove it but forgot